<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addLeave</class>
 <widget class="QDialog" name="addLeave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Leave</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Leave</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deLeaveDate"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>end</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deEndDate"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>pay type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxPayType">
            <item>
             <property name="text">
              <string>Full Salary </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Half Salary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NO Salary</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Employees</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListView" name="lvEmployee"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
            <item>
             <widget class="QPushButton" name="pbtnAddEmp">
              <property name="text">
               <string>Add Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtnRemEmp">
              <property name="text">
               <string>Remove Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtnSelectAll">
              <property name="text">
               <string>Select All</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>addLeave</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
